crates/ruff_linter/src/rules/pyupgrade/rules/deprecated_import.rs

```diff
@@ -342,13 +347,18 @@ const TYPING_EXTENSIONS_TO_TYPING_312: &[&str] = &[
     "SupportsFloat",
     "SupportsInt",
     "SupportsRound",
-    "TypedDict",
+    "TypeAliasType",
     "Unpack",
     // Introduced in Python 3.11, but `typing_extensions` backports the `frozen_default` argument,
     // which was introduced in Python 3.12.
     "dataclass_transform",
+    "get_original_bases",
```

Comment:
Ah, this is in the `types` module in Python 3.12, not `typing`

Reply:
you can blame me for that one, i reviewed and merged the PR adding it...

Reply:
Oops, thanks!